Docs/About.md

```diff
@@ -19,9 +19,9 @@ The following Swift packages are used in BluePic:
 - [Kitura-CouchDB](https://github.com/IBM-Swift/Kitura-CouchDB)
 - [Kitura](https://github.com/IBM-Swift/Kitura.git)
 - [Cloud Configuration](https://github.com/IBM-Swift/CloudConfiguration.git)
-- [Swift SDK for Bluemix Object Storage Service](https://github.com/ibm-bluemix-mobile-services/bluemix-objectstorage-swift-sdk.git)
+- [Swift SDK for IBM Cloud Object Storage Service](https://github.com/ibm-bluemix-mobile-services/bluemix-objectstorage-swift-sdk.git)
 - [Kitura Credentials plugin for the App ID service](https://github.com/ibm-cloud-security/appid-serversdk-swift)
-- [Swift SDK for Bluemix Push Notifications Service](https://github.com/ibm-bluemix-mobile-services/bms-pushnotifications-serversdk-swift.git)
+- [Swift SDK for IBM Cloud Push Notifications Service](https://github.com/ibm-bluemix-mobile-services/bms-pushnotifications-serversdk-swift.git)
 
 ## Architecture
```

Comment:
How about we replace the current architectural image we have in the About.md docs with the one at https://developer.ibm.com/code/patterns/apply-cognitive-to-mobile-images-on-the-go/?